### ISSUE TITLE
Erroneous “Failed to refresh” popup message on save

### ISSUE THREAD

**Issue by aviflax** [mauricioszabo/atom-chlorine#30]
When I’m connected to a socket REPL, every time I save a .clj file in my project I get a _Failed to refresh_ popup message, saying there’s a syntax error in one of my files. The thing is, I’m pretty sure that’s not true; when I load the files manually from the REPL in my terminal they compile just fine.

This is with the following package settings:

* full namespace refresh with clojure.tools.namespace
* refresh namespaces on save

Full contents of the popup:

```edn
{:ex #error {:via [{:type clojure.lang.Compiler$CompilerException, :message "Syntax error compiling var at (dsl.clj:31:116).", :data {:clojure.error/phase :compile-syntax-check, :clojure.error/line 31, :clojure.error/column 116, :clojure.error/source "dsl.clj", :clojure.error/symbol var}, :at [clojure.lang.Compiler analyzeSeq "Compiler.java" 7114]} {:type java.lang.RuntimeException, :message "Unable to resolve var: clojure.test/*load-tests* in this context", :at [clojure.lang.Util runtimeException "Util.java" 221]}], :trace [[clojure.lang.Util runtimeException "Util.java" 221] [clojure.lang.Compiler$TheVarExpr$Parser parse "Compiler.java" 720] [clojure.lang.Compiler analyzeSeq "Compiler.java" 7106] [clojure.lang.Compiler analyze "Compiler.java" 6789] [clojure.lang.Compiler analyze "Compiler.java" 6745] [clojure.lang.Compiler$InvokeExpr parse "Compiler.java" 3888] [clojure.lang.Compiler analyzeSeq "Compiler.java" 7108] [clojure.lang.Compiler analyze {:repl-tooling/... (unrepl.repl$JheDmiOxCQTNBrUuDsZnDvb59lc/fetch :G__9644)}] {:repl-tooling/... (unrepl.repl$JheDmiOxCQTNBrUuDsZnDvb59lc/fetch :G__9645)}], {:repl-tooling/... nil} {:repl-tooling/... (unrepl.repl$JheDmiOxCQTNBrUuDsZnDvb59lc/fetch :G__9646)}}, {:repl-tooling/... nil} {:repl-tooling/... (unrepl.repl$JheDmiOxCQTNBrUuDsZnDvb59lc/fetch :G__9647)}}
```

And an image:

<img width="491" alt="screen shot 2019-01-10 at 16 53 58" src="https://user-images.githubusercontent.com/141844/50999531-5ad71200-14f8-11e9-866c-09d04217cc70.png">

I’m using:

* Atom 1.34.0
* Chlorine 0.0.7
* Clojure 1.10.0
* Java 11
* MacOS 10.13

Thanks for all the great work on Chlorine! Please let me know if there’s any way I can help reproduce this. I could point you to the code I’m loading, for example. Thanks!

**Comment by seancorfield** [comment]
Looks like this is already fixed on the `goto-var` branch https://github.com/mauricioszabo/atom-chlorine/commit/6797ab25ba0e6dcc81ec77b940dca6e552224d72

**Comment by mauricioszabo** [comment]
Yes, it is already fixed. The only issue is that in the past, to support some code I used little hacks that are unnecessary right now, so I'm fixed these issues before publishing a new version

**Comment by aviflax** [comment]
Great!

**Comment by mauricioszabo** [comment]
Fixed on 0.0.8
